packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -147,6 +153,8 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
       provisionedConcurrencyConfig: this.determineProvisionedConcurrency(props),
     });
 
+    this.provisionedConcurrency = props.provisionedConcurrentExecutions ? true : false;
```

Comment:
I'm unsure we should force customers to set a starting PC to enable autoscaling. We should test what happens if you set no PC in CFN and then autoscaling minimum to like...5. Does Autoscaling start provisioning concurrency? If yes, customers may not want to set a starting amount.

Reply:
Yep, autoscaling works fine without a starting PC. Thanks for pointing this out.